sherpa-onnx/python/sherpa_onnx/__init__.py

```diff
@@ -16,6 +16,7 @@
     OfflineTtsVitsModelConfig,
     OfflineZipformerAudioTaggingModelConfig,
     OnlineStream,
+    ProviderConfig,
```

Comment:
Could you remove it?

I think it is not used outside of online_recognizer.py and keyword_spotter.py.

Reply:
yes I think keyword spotter was not able to import it, without adding providerconfig at __init__ i think,
Not very sure though, I will remove it, I the tests fails later, I will add it back